src/components/Charts/MetricChart.tsx

```diff
@@ -21,37 +21,35 @@ export default function MetricChart({
   height?: number;
 }) {
   const projection = projections.primary;
-  if (!projection) {
+  if (projection == null || projections.getMetricValue(metric) == null) {
```

Comment:
this will catch undefined too do you mean ===

Reply:
Again, half intentional but I went ahead and made it more strict.  Seems to still work.